server/api.js

```diff
@@ -58,8 +58,7 @@ const fetchAuthorizationCode = async (userId, clientAccessToken) => {
     'credentials:read,credentials:refresh,credentials:write', // needed to enable add/refresh/authenticate credentials
     'payment:read,transfer:read,transfer:execute', // needed for executing payment requests - creating a transfers
     'link-session:read', // reading sessions
-    'user-report-bundles:read,account-verification-reports:write', // needed for Account Check fetching accounts call
-    'provider-consents:read,provider-consents:write', // TODO:
```

Comment:
Deleting these will break the balance check flow (account check permanent user thing).

Reply:
Did we update their names?

<img width="709" alt="Screen Shot 2022-10-26 at 3 34 41 PM" src="https://user-images.githubusercontent.com/29574724/198040050-3cf66042-e7ab-48ed-b4b4-c3e089f3cfe0.png">

Reply:
Ah my bad, it's probably my app